tools/Dashboard/DevHome.Dashboard/ViewModels/WidgetViewModel.cs

```diff
@@ -124,8 +124,9 @@ private async Task RenderWidgetFrameworkElementAsync()
                 return;
             }
 
-            Log.Logger()?.ReportDebug("WidgetViewModel", $"cardTemplate = {cardTemplate}");
-            Log.Logger()?.ReportDebug("WidgetViewModel", $"cardData = {cardData}");
+            // Uncomment for extra debugging output

```

Comment:
Usually we don't leave commented out code in since they are in the history if we want them back for local debugging, but I'm OK with it here if it is commonly used for troubleshooting dashboard bugs.